components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/ServiceURL.java

```diff
@@ -37,7 +37,7 @@
      *
      * @return String of the host name.
      */
-    String getHostName();
+    String getProxyHostName();
```

Comment:
This component did not go in a released product version yet right? We have changed the interface method name here.

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3111/commits/beb53f7154b1bbf61aeb6b1130af4fc849b4b681

Reply:
The framework version with this interface is not yet available in the released product.